core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java

```diff
@@ -102,6 +112,67 @@ private void testInstall(String javaVersion)
         }
     }
 
+    @Test
+    public void testRpmContents()
```

Comment:
We need to verify the symlinks to `shared` for JARs

Reply:
Done.